dags/telemetry_aggregates.py

```diff
@@ -0,0 +1,180 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
+from utils.gcp import gke_command
+from utils.mozetl import mozetl_envvar
+from utils.status import register_status
+
+common = {
+    "depends_on_past": True,
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 3,
+    "retry_delay": timedelta(minutes=30),
+}
+
+default_args = common.copy()
```

Comment:
How about renaming `default_args` to `default_args_prerelease` here and similarly below?